docs/application/dotnet/guides/connectivity/stc.md

```diff
@@ -0,0 +1,166 @@
+# Smart Traffic Control (STC)
+
+
+Smart Traffic Control (STC) provides extensible packet-level control services, including per-app data usage, total data quota, and background app's data saving.
+
+The main features of the Tizen.Network.Stc namespace include:
+
+- Retrieve Data Usage for System
+
+  You can [retrieve the network data consumed by the system](#retrieve-data-usage-for-system).
+
+- Retrieve Data Usage for Applications
+
+  You can [retrieve the network data consumed by the applications](#retrieve-data-usage-for-applications).
+
+
+> **Note**  
+> You can test the STC functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.
```

Comment:
Please insert one empty line with > between line 17 and line 18.

\> \*\*Note**
\>
\> You can test the STC functionality on a target device only. The [emulator](../../../tizen-studio/common-tools/emulator.md) does not support this feature.

Reply:
done